grammar.js

```diff
@@ -58,7 +58,7 @@ module.exports = grammar({
                     ),
                     seq($.super, ".", $.id),
                     seq($.super, "[", $.expr, "]"),
-                    seq($.expr, "(", optional($.args), ")"),
+                    seq($.expr, "(", optional($.args), ")", optional("tailstrict")),
```

Comment:
can we make this a new node?

so make a new `tailstrict: ($) => "tailstrict"` and then do `optiona($.tailstrict)` here? This way it will show up in the resulting tree.

Reply:
ok